manager/testdata/notebook/README.md

```diff
@@ -0,0 +1,347 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: paysim-secret
+type: Opaque
+stringData:
+  access_key: "${ACCESS_KEY}"
+  secret_key: "${SECRET_KEY}"
+EOF
+```
+
+Then, register the data assets in the catalog. Replace the values for `endpoint`, `bucket` and `object_key` with values from the object storage service that you used and run:
+
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: orig-paysim-csv
+spec:
+  secretRef: 
+    name: paysim-secret
+  details:
+    dataFormat: csv
+    connection:
+      name: s3
+      s3:
+        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+        bucket: "demo"
+        object_key: "PS_20174392719_1491204439457_log.csv"
+  metadata:
+    name: Synthetic Financial Datasets For Fraud Detection
+    geography: theshire 
+    tags:
+      finance: true
+    columns:
+      - name: nameOrig
+        tags:
+          PII: true
+      - name: oldbalanceOrg
+        tags:
+          sensitive: true
+      - name: newbalanceOrig
+        tags:
+          sensitive: true
+EOF
+```
+```yaml
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: new-paysim-parquet
+spec:
+  secretRef:
+    name: paysim-secret
+  details:
+    dataFormat: parquet
+    connection:
+      name: s3
+      s3:
+        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+        bucket: "bucket1"
+        object_key: ""
+  metadata:
+    name: Synthetic Financial Datasets For Fraud Detection
+    geography: theshire
+    tags:
+      finance: true
+    columns:
+      - name: nameOrig
+        tags:
+          PII: true
+      - name: oldbalanceOrg
+        tags:
+          sensitive: true
+      - name: newbalanceOrig
+        tags:
+          sensitive: true
+EOF
+```
+The assets are now registered in the catalog. The identifier of the original asset is `fybrik-notebook-sample/orig-paysim-csv` (i.e. `<namespace>/<name>`) and the identifier of the new asset is `fybrik-notebook-sample/new-paysim-parquet`. You will use these names in the `FybrikApplication` later.
+
+Notice the `metadata` field above. It specifies the dataset geography and tags. These attributes can later be used in policies.
+
+For example, in the yaml above, the `geography` is set to `theshire`, you need make sure it is same with the region of your fybrik control plane, you can get the information with the below command:
+
+```shell
+kubectl get configmap cluster-metadata -n fybrik-system -o 'jsonpath={.data.Region}'
+```
+
+[Quick Start](../get-started/quickstart.md) installs a fybrik control plane with the region `theshire` by default. If you change it or the `geography` in the yaml above, a [copy module](https://github.com/fybrik/mover) will be required by the policies, but we do not install any copy module in the [Quick Start](../get-started/quickstart.md).
+
+## Define data access policies
```

Comment:
Need to point out that these are for reading only, and currently there are no policies defined for write